src/Sentry/DisabledMetricAggregator.cs

```diff
@@ -0,0 +1,28 @@
+namespace Sentry;
+
+internal class DisabledMetricAggregator : IMetricAggregator
+{
+    public void Increment(string key, double value = 1, MeasurementUnit? unit = null, IDictionary<string, string>? tags = null,
```

Comment:
Replying here for no good reason, but can `MeasurementUnit` hold an arbitrary non defined unit? What is this, is it an enum?